example/ui.gui_script

```diff
@@ -122,7 +122,8 @@ function init(self)
   
   if ironsource then
     log(self, "ironsource init with app_key: " .. YOUR_APP_KEY)
-    ironsource.init(YOUR_APP_KEY, GDPR_CONSENT)
+    ironsource.set_consent(GDPR_CONSENT)
```

Comment:
[https://developers.ironsrc.com/ironsource-mobile/android/advanced-settings/#step-1](Instruction) says:
`It’s recommended to set the API prior to SDK Initialization.`
I missed this earlier!
May be add comment in example?

Reply:
I've removed the whole set_consent() call from the example, it's not really needed here.